engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-publish.html

```diff
@@ -25,6 +25,18 @@ <h3 translate>Version Details</h3>
               tabindex="1">
     </textarea>
   </div>
+  <div bst-form-group label="{{ 'Force Metadata Regeneration' | translate }}">
+    
+    <input id="force_metadata_regeneration"
+           name="force_metadata_regeneration"
+           ng-model="version.force_metadata_regeneration"
+           type="checkbox"/>
+    <i class="fa fa-question-circle"
+       uib-tooltip="{{ 'Forces metadata regeneration on yum repos even if the contents of the repo have not changed' | translate }}"
+       tooltip-animation="false"
+       tooltip-append-to-body="true">
+    </i>
```

Comment:
@waldenraines any idea how to get this tooltip next to the [label](https://github.com/Katello/katello/pull/6597/files#diff-3fb1ccd1f321ef57e95f49ee64506a5eR28)?